### PR TITLE
Remove IE<= 10 CSS hacks

### PR DESCRIPTION
**Pull request by AnaelMobilia**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | none

Fix errors on CSS files (comments, bad values, unexisting files...)
Remove IE <= 10 specific CSS + JS

### CODE REVIEW

File: css/print.css [hunk 1]
```diff
@@ -980,8 +980,8 @@ ul#onglet span.fake {
     /*css hack for IE*/
    position: absolute;
    bottom: auto;
-   top: expression((parseInt(document.documentElement.clientHeight)+parseInt(document.documentElement.scrollTop)-42)+'px');
-   width: expression(parseInt(document.documentElement.clientWidth)+'px');
+   /*top: expression((parseInt(document.documentElement.clientHeight)+parseInt(document.documentElement.scrollTop)-42)+'px');
```
Comment (trasher) [inline css/print.css]: This should be dropped, not commented ;)

File: css/styles.css [hunk 2]
```diff
@@ -1152,12 +1152,12 @@ td.rotate {
   -ms-transform:rotate(-70deg); /* IE 9 */
   width: 10%;
   height: 80px;
-  <!--[if lte IE 7]>
-   filter:  progid:DXImageTransform.Microsoft.BasicImage(rotation=0.083);  /* IE6,IE7 */
-  <![endif]-->
-  <!--[if lte IE 8]>
-   -ms-filter: "progid:DXImageTransform.Microsoft.BasicImage(rotation=0.083)"; /* IE8 */
-  <![endif]-->
+  /*<!--[if lte IE 7]>*/
```
Comment (trasher) [inline css/styles.css]: Same drop

**Comment by AnaelMobilia** [comment]
Maybe we need to think to use a GUI framework ?
See #3392

File: css/jstree-glpi.css [hunk 3]
```diff
@@ -329,8 +329,6 @@
   opacity: 0.8;
   filter: url("data:image/svg+xml;utf8,<svg xmlns=\'http://www.w3.org/2000/svg\'><filter id=\'jstree-grayscale\'><feColorMatrix type=\'matrix\' values=\'0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0 0 0 1 0\'/></filter></svg>#jstree-grayscale");
   /* Firefox 10+ */
-  filter: gray;
-  /* IE6-9 */
   -webkit-filter: grayscale(100%);
```
Comment (trasher) [inline css/jstree-glpi.css]: According to MDN, `filter: grayscale(100%);` can be used for Firefox. `-webkit` must be kept for edge.

Comment (AnaelMobilia) [reply css/jstree-glpi.css]: Be warn that on this case, the developper write code and the line below the comments ;-)
Else IE6-9 have the webkit rendering engine 😨

Comment (trasher) [reply css/jstree-glpi.css]: Yes, I've understand that :D

So, `filter: url()` was for Firefox, but this does not seems needed right now.

**Comment by orthagh** [comment]
@AnaelMobilia 
Do you mind if i push some additional commits to this pr ? i want to clean more the common.js part and fix some edge issues.

**Comment by AnaelMobilia** [comment]
@orthagh No problems :-)
Thanks for asking !

**Pull request merged**